File: tfr1/summat/split.java
public class split {
    public static char[][] split(char[] s) {
        char[][] a = new char[5][3]; 
        for (int i = 0; i < 3; i++) {
           a[0][i] = s[i]; 
        }
        for (int i = 3; i < 6; i++) {
            a[1][i] = s[i];
        }
        return a;
    }
    public static void main(String[] args) {
        char[] s = {'o', 's', 'h', '\t','6','9',' ','l','e','e','d','s'};
        //char[] split_s = split(s);
        char[][] b = split(s);
        for (int i = 0; i < b.length; i++)
            System.out.print(b[i]);
        System.out.println();
    }
}

File: tfr2/hd/h2/d2/split.c
#include <stdio.h>

char** split(const char* s, int *n) {
    int i = 0;
    while (i != '\t') {
        char fylki[i] = i;
        printf("%c", i);
    return fylki;
}

int main() {
    char *s = "oskar\telskar\tleeds"; // fylla inn
    int n = 3;
    char** split_s = split(s, &n);
    // ... restin pls
}
